modules/chewbbaca/main.nf

```diff
@@ -69,23 +70,24 @@ process chewbbaca_create_batch_list {
 
 process chewbbaca_split_results {
   label "process_low"
-  tag "${assembly.simpleName}"
+  tag "${sampleName}"
   publishDir "${params.publishDir}", 
     mode: params.publishDirMode, 
     overwrite: params.publishDirOverwrite
 
   input:
+    each sampleName
     path input
 
   output:
-    path("${output}")
+    tuple val(sampleName), path("${output}")
 
   script:
     id = "${input.simpleName}"
```

Comment:
Do we need line 86?

Reply:
simpleName is a nextflow function that strips extension and filepath base. Regardless, it should be removed as it is old code.